Modules/Filtering/ImageFeature/test/CMakeLists.txt

```diff
@@ -34,7 +34,10 @@ set(TEMP ${ITK_TEST_OUTPUT_DIR})
 itk_add_test(NAME itkZeroCrossingBasedEdgeDetectionImageFilterTest
       COMMAND ITKImageFeatureTestDriver itkZeroCrossingBasedEdgeDetectionImageFilterTest 1.0 0.01)
 itk_add_test(NAME itkLaplacianImageFilterTest
-      COMMAND ITKImageFeatureTestDriver itkLaplacianImageFilterTest)
+      COMMAND ITKImageFeatureTestDriver
+    --compare-MD5 ${ITK_TEST_OUTPUT_DIR}/itkLaplacianImageFilterTest.png
```

Comment:
Later debugging of a failed test is harder if only a file hash is provided. How big is the file?

Reply:
Files will be small.  Changed to the raw images in 155b729.